node-runner-cli/commands/authcommand.py

```diff
@@ -0,0 +1,53 @@
+from argparse import ArgumentParser
```

Comment:
All the files in commands folder are existing commands and they were refactored from single file to individual files in commands module for easy maintainence

Reply:
I like this change :)